### ISSUE TITLE
Cannot change standards

### ISSUE THREAD

**Issue by wolffens** [squizlabs/HTML_CodeSniffer#203]
Cannot change standards, like Section508,WCAG2A,WCAG2AA or WCAG2AAA on https://www.nijmegen.nl

**Comment by ironikart** [comment]
You have a CSS rule in the main.css file that is hiding all <select> elements. HTMLCS gets injected to your page and thus inherits any css that you are using. We try to avoid conflict by styling all elements with namespaced id and selectors but unfortunately nothing we can do about styling of element selectors.

**Comment by wolffens** [comment]
Thank you for figuring out!
